lib/request-handler.js

```diff
@@ -89,31 +73,23 @@ function getBadgeMaxAge(handlerOptions, queryParams) {
 // (undesirable and hard to debug).
 //
 // Pass just the handler function as shorthand.
-function handleRequest(handlerOptions) {
+function handleRequest(cacheHeaderConfig, handlerOptions) {
```

Comment:
Instead of reading from the environment, inject the config (via `BaseService` and `LegacyService`, in turn via `server.js`).